server/src/main/resources/rules/rules.js

```diff
@@ -1311,8 +1354,29 @@ function createComplianceTracker(consumer, id) {
 
                 guest_limit: function (currentStackValue, poolValue, pool, quantity) {
                     return -1; //Value doesn't matter, just need it to be enforced
-                }
+                },
 
+                /**
+                 *  AddOns is accumulated by adding each pool value to
+                 *  a list of addon strings. Each pool value is a , separated
```

Comment:
Please change "," to "comma".  It took me a few seconds to figure out that a word wasn't missing here.